draft-thomson-webpush-protocol.xml

```diff
@@ -425,21 +361,16 @@ Cache-Control: max-age=600
       </figure>
       <t>
         A push service MAY generate a 413 (Payload Too Large) status code in
-        response to PUT requests that include an entity body that is too large.
+        response to POST requests that include an entity body that is too large.
```

Comment:
Maybe drop the word "POST" here in anticipation of doing both POST and PUT later.